src/ray/common/memory_monitor.cc

```diff
@@ -118,7 +118,17 @@ std::tuple<int64_t, int64_t> MemoryMonitor::GetCGroupMemoryBytes() {
             (total_bytes != kNull && used_bytes != kNull));
   if (total_bytes != kNull) {
     RAY_CHECK_GT(used_bytes, 0);
-    RAY_CHECK_GT(total_bytes, used_bytes);
+    if (used_bytes <= 0) {
+      RAY_LOG_EVERY_MS(WARNING, kLogIntervalMs)
```

Comment:
in what circumstance the used_bytes will be negative?

Reply:
if we can't read / read something weird from the OS?
I changed so log and set used to 0 in these cases